src/Container.js

```diff
@@ -155,33 +155,37 @@ class Container extends React.Component{
   }
 
 
-  handleExportData =(_) => {
+  handleExportData = (_) => {
     let cachedDataJSON = (JSON.parse(localStorage.getItem("matchData")));
     let cachedDataCSV = "";
 
-    if (cachedDataJSON != null){
-      for (let i = 0; i < cachedDataJSON.length; i++){
-        for (let e = 0; e < cachedDataJSON[i].length; e++){
+    if (cachedDataJSON != null) {
+      for (let i = 0; i < cachedDataJSON.length; i++) {
+        for (let e = 0; e < cachedDataJSON[i].length; e++) {
           cachedDataCSV += cachedDataJSON[i][e][1] + ","
         }
         cachedDataCSV += "\n"
       }
     }
 
-    let file = new Blob([cachedDataCSV], {type: "text/csv"})
+    let file = new Blob([cachedDataCSV], { type: "text/csv" })
     let blobURL = window.URL.createObjectURL(file)
 
     const anchor = document.createElement('a');
     anchor.href = blobURL
     anchor.target = "_blank"
     anchor.download = "matchData.csv"
- 
+
     anchor.click()
- 
+
     URL.revokeObjectURL(blobURL);
   }
 
-  render () {
+  sliderNumbers = () => {
+    return "1 ‎ ‎ ‎ ‎ ‎ ‎ ‎ 2 ‎ ‎ ‎ ‎ ‎ ‎ ‎ 3 ‎ ‎ ‎ ‎ ‎ ‎ ‎ 4 ‎ ‎ ‎ ‎ ‎ ‎ ‎ 5";
```

Comment:
Structuring our code like this might make it more difficult to add to our scouting form in future years as we are relying on hard coded whitespaces instead of more dynamic layouts

Reply:
To expand on this further, I think another way that you could do this is by using a dynamically sized flexbox based on the number of slider numbers, so that we put less bulk into Container.js and also to rely less on hardcoded values. Ideally this flexbox would be in the slider widget class along with other elements pertaining to these sliders.